### PR TITLE
Portfolio: finish mobile version

### PR DESCRIPTION
**Pull request by momotsow**
For this Portfolio project I have added 2 sections as per the requirement namely:

- Work Section
- About Myself Section

I have chosen Figma Template 5 for this exercise.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -24,5 +24,146 @@ <h1 class="headline">Charlie Giorgania</h1>
         </div>
     </section>
     <div class="header-img2"></div>
+    <section class="projects">
+
+        <h1 class="tittle t2">Projects</h1>
```
Comment (leonardodiasb) [inline index.html]: - [ ] Your project already has a main title with the `<h1>` tag. Please follow the headings hierarchy and use a proper `<hx>` tag for this section title. [Read this article for more information](https://www.reliablesoft.net/h1-tag/)

File: styles.css [hunk 2]
```diff
@@ -69,11 +74,197 @@ button {
...
+    font-weight: 300;
+    font-size: 20px;
+    line-height: 80%;
+    margin: 0;
+    padding: 20px 0 16px 20px;
+  }
+
+  .t2 {
+    margin-left: 16px;
+  }
```
Comment (leonardodiasb) [inline styles.css:160]: - [ ] This title does not have the same font-family, font-size, and color as the Figma template. Please add these attributes to make it look more like the template.

File: index.html [hunk 3]
```diff
@@ -24,5 +24,146 @@ <h1 class="headline">Charlie Giorgania</h1>
...
+                <button class="project-link">HTML/CSS</button>
+                <button class="project-link">Ruby on Rails</button>
+                <button class="project-link">JavaScript</button>
+            </div>
+            <button class="project-button">See this project<span><img src="./images/ic_arrow_right.png" alt=""></span></button>
+        </article>
+    </section>
+    <section class="about-section container">
+
+        <h1 class="tittle">About</h1>
```
Comment (leonardodiasb) [inline index.html]: - [ ] Your project already has a main title with the `<h1>` tag. Please follow the headings hierarchy and use a proper `<hx>` tag for this section title. [Read this article for more information](https://www.reliablesoft.net/h1-tag/)
 - [ ] This title does not have the same font-family, font-size, and color as the Figma template. Please add these attributes to make it look more like the template.

File: index.html [hunk 4]
```diff
@@ -24,5 +24,146 @@ <h1 class="headline">Charlie Giorgania</h1>
...
+                <ul class="profile-list">
+                    <li>Codekit</li>
+                    <li>Codepen</li>
+                    <li>GitHub</li>
+                    <li>GitLAB</li>
+                    <li>Terminal</li>
+                </ul>
+                <div class="bg-img3"></div>
+            </article>
+        </div>
```
Comment (leonardodiasb) [inline index.html:170]: - [ ] This is a list of technology categories, and an `<ul>` tag with a `<li>` tag for each category would be better appropriate in this situation for semantic reasons. Kindly change the HTML tags.

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @momotsow, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

 - Great UI design following the Figma template almost perfectly. ✔️ ✔️ ✔️
 - No linter errors. ✔️ 

## Aspects to improve ♻️

- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by momotsow** [comment]
Hi @leonardodiasb,

I need clarity on the last review.
In my own understanding of you review- you are saying I should use an unordered list `<ul><li></li></ul>`

I believe that's what I did as I created 4 Articles/cards with an unordered list inside as per the Figma design
image

File: index.html (hunk 1 above)
Comment (mgmediaweb) [reply index.html]: Good job following the advice of the previous reviewer

File: index.html [hunk 5]
```diff
@@ -24,5 +24,146 @@ <h1 class="headline">Charlie Giorgania</h1>
...
+
+        <article class="project-container">
+
+            <img class="img" src="./images/Rectangle6.png" alt="Project name goes here">
+            <h4 class="project-tittle">Project name goes here</h4>
+            
+            <div class="project-skills containe">
+                <button class="project-link">HTML/CSS</button>
+                <button class="project-link">Ruby on Rails</button>
+                <button class="project-link">JavaScript</button>
```
Comment (mgmediaweb) [inline index.html]: kindly, I suggest following the advice of the previous reviewer and use UL/LI lists to organize languages

File: index.html [hunk 6]
```diff
@@ -24,5 +24,146 @@ <h1 class="headline">Charlie Giorgania</h1>
...
+
+        <article  class="project-container">
+
+            <img class="img" src="./images/Rectangle5.png" alt="Project name goes here">
+            <h4 class="project-tittle">Project name goes here</h4>
+            
+            <div class="project-skills containe">
+                <button class="project-link">HTML/CSS</button>
+                <button class="project-link">Ruby on Rails</button>
+                <button class="project-link">JavaScript</button>
```
Comment (mgmediaweb) [inline index.html]: kindly, I suggest following the advice of the previous reviewer and use UL/LI lists to organize languages

**Review by mgmediaweb — CHANGES_REQUESTED** [review]
Hi @momotsow , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

almost4

 ## Highlights 💯 
- Clean and tidy code. ✔️ 
- Good job following the design. ✔️ 
- Good job following the advice of the previous reviewer ✔️ 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Nyame-Wolf — APPROVED** [review]
# STATUS: APPROVED :heavy_check_mark: :green_circle: 

Hi @momotsow  :smile: , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
giphy


## Highlights :sparkles:

## Optional suggestions :ballot_box_with_check: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

APPROVED! 💯 💯 💯

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me, @Nyame-Wolf in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**